ranch/ranch.go

```diff
@@ -165,6 +166,10 @@ func (r *Ranch) Acquire(rType, state, dest, owner, requestID string) (*crds.Reso
 			}
 			// Deleting this request since it has been fulfilled
 			if requestID != "" {
+				if createdTime, err = r.requestMgr.GetCreatedAt(ts, requestID); err != nil {
+					logger.WithError(err).Errorf("Error occurred when getting the created time")
+					return err
```

Comment:
Does it make sense to return the error here? I'm not sure we should make `Acquire` fail here, especially given that we've already updated the resource to give ownership.

Reply:
Updated.